extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -272,6 +272,7 @@ describe('amp-ad-3p-impl', () => {
         iframe.style.width = '300px';
         iframe.style.height = '200px';
         iframe.style.display = 'block';
+        iframe.style.minHeight = '0px';
```

Comment:
what does this accomplish? isn't the iframe's default minHeight 0 already like other elements?